finder-app/finder_bk2.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
```

Comment:
@Sharath-Jonnala see previous comment about using revision control to save history.